driver/normalizer/annotation.go

```diff
@@ -119,9 +81,9 @@ var AnnotationRules = On(Any).Self(
 		On(HasInternalType(pyast.Div)).Roles(OpDivide),
 		On(HasInternalType(pyast.Mod)).Roles(OpMod),
 		// TODO: currently without mapping in the UAST
-		//On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide),
-		//On(HasInternalType(pyast.Pow)).Roles(???),
-		//On(HasInternalType(pyast.MatMult)).Roles(???),
+		On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide, Incomplete),
+		On(HasInternalType(pyast.Pow)).Roles(OpMultiply, Incomplete),
```

Comment:
Yes, it was not totally clear from the original phrasing, PTAL: https://github.com/bblfsh/documentation/pull/74